src/Scrabble/Search.hs

```diff
@@ -54,6 +56,22 @@ containsAll :: String -> Search
 containsAll s1 s2 = fst $ foldl' f (True, ups s2) (ups s1) where
   f (b,s) c = if elem c s then (b, delete c s) else (False,s)
 
+-- | Like the above, but if it encounters a character in in s1 not in
+--   s2, it deletes a blank and tries again before returning false
+containsAllWithBlanks :: String -> Search
```

Comment:
Second easiest thing to do would just be to move them into Move.hs. Just a quick copy and paste and now those functions are local to the only place where they're used.